src/GraphQLParser/Visitors/PrintContextExtensions.cs

```diff
@@ -15,27 +15,34 @@ public static class PrintContextExtensions
     /// using <see cref="CancellationToken"/> from it.
     /// </summary>
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public static ValueTask WriteAsync<TContext>(this TContext context, ROM value)
+    public static async ValueTask WriteAsync<TContext>(this TContext context, ROM value)
         where TContext : IPrintContext
     {
+        if (value.Length == 0)
+            return;
```

Comment:
I suggest `return default;` and remove `async` and remove `await task`.  Whatever you want though.

Reply:
Ah, I forgot to revert it after some of my previous changes.

Reply:
Done